xml/en/download.xml

```diff
@@ -42,7 +42,19 @@ Read-only Mercurial repositories:
 code: <literal>http://hg.nginx.org/nginx</literal>
 </listitem>
 <listitem>
-site: <literal>http://hg.nginx.org/nginx.org</literal>
+website: <literal>http://hg.nginx.org/nginx.org</literal>
+(mirror of GitHub repository)
+</listitem>
+
+</list>
+</para>
+
+<para>
+GitHub repositories:
```

Comment:
If we're listing Mercurial and GitHub repos for 'website' then shouldn't we also list them both for 'code'?

Something like this, for completeness:

> ~Read-only~ Mercurial repositories:
> 
> • code: http://hg.nginx.org/nginx (read-only)
> • website: http://hg.nginx.org/nginx.org (mirror of GitHub repository)
> 
> GitHub repositories:
> 
> • code: http://hg.nginx.org/nginx (mirror of Mercurial repository)
> • website: https://github.com/nginx/nginx.org

Reply:
Good point, thanks, I'll update the patch accordingly